package.json

```diff
@@ -164,14 +164,14 @@
 		"@wordpress/babel-preset-default": "4.10.0",
 		"@wordpress/browserslist-config": "2.6.0",
 		"@wordpress/custom-templated-path-webpack-plugin": "1.6.0",
-		"@wordpress/jest-preset-default": "5.5.0",
```

Comment:
Potentially naive question: Would downgrading `@wordpress/jest-preset-default` do the task without the other modifications?

Reply:
@psealock thats not naive, except I'd rather move away from the preset for now because the setup files it runs crash in a `node` environment test (which will be crucial if we want to keep improving performance). It is my plan to raise an issue about this in Gutenberg.